README.rst

```diff
@@ -74,13 +70,25 @@ You could then activate the environment created at ``myEnv``.
 
 Using a different environment management tool
 """""""""""""""""""""""""""""""""""""""""""""
-The InVEST Makefile uses ``virtualenv`` to set up an environment, but this is
-not the only `environment management tool out there
-<https://packaging.python.org/tutorials/installing-packages/#creating-virtual-environments>`_.
-You may elect to manage your virtual environment a different way, independent
-of ``make env``.  The only requirement for the build process is that the required
-tools are available on your PATH and the required python packages can be imported.
+You may of course choose to manage your own virtual environment without using the Makefile.
+
+We suggest using ``conda`` or ``mamba`` and ``conda-forge``.
+
+``requirements.txt``, ``requirements-dev.txt`` and ``requirements-docs.txt`` list the python
+dependencies needed.
+
+Installing ``natcap.invest`` from local source code
+"""""""""""""""""""""""""""""""""""""""""""""""""""
+From an activated virtual environment::
 
+    $ make install
+
```

Comment:
I would add a note here about the need to uninstall before reinstalling. Something along these lines:

```
To update your local installation after making changes to source code::
    $ pip uninstall natcap-invest
    $ make install
```
This'll get you unstuck if you (intentionally or unintentionally) use `make install`, and it underscores why an editable install (described next) is so compelling.